### PR TITLE
Updating the clear filters button for tablet view.

### LINKED ISSUE #99: Clear Filters doesn’t look like a button on phone/tablet viewports
@chrismulholland picked this up in testing. I've taken a look and it does look pretty stretched / non-button-y at various mobile sizes. @ricardoom, can you check it out and make a recommendation?

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes #99 

Check it out on dev for tablet view.

### CODE REVIEW

File: src/client/styles/components/_bookFilters.scss [hunk 1]
```diff
@@ -103,11 +103,17 @@
   .clear-button {
     width: 88%;
     text-align: right;
-    border-radius: 0px;
+    border-radius: 5px;
```
Comment (rhernand3z) [inline src/client/styles/components/_bookFilters.scss]: there is a mixin for this to tackle cross-browser compatibility...

**Comment by rhernand3z** [comment]
When running locally, I see the following issues on both FF and Chrome:

1) When the tag text is long, the X is right next to it...
screen shot 2017-11-14 at 3 38 02 pm

2) On viewport 965px width, it does not adapt to a 2 column layout. I see a huge white space on the right (see screenshot)
screen shot 2017-11-14 at 3 38 25 pm

**Comment by EdwinGuzman** [comment]
Right, the breakpoint is exactly 965px, and we're not doing "[width size] + 1", so that only happens for that exact width.

**Comment by rhernand3z** [comment]
Shouldn't the filters auto fit into columns up until the grid splits into 1/3 | 3/4 ? I assume we should account up until that split happens?

**Comment by chrismulholland** [comment]
image

**Comment by EdwinGuzman** [comment]
Cool. The 'X' being too close to the filter text is not directly related to this issue so I'm going to create another issue. Thanks.

**Comment by chrismulholland** [comment]
is the multi-column button layout supposed to appear in all browsers at certain widths?  If so, I don't see it in Chrome.  Firefox, yes.

**Comment by EdwinGuzman** [comment]
It's suppose to appear in viewports between 483px and 966px.

**Comment by EdwinGuzman** [comment]
@rhernand3z @chrismulholland check dev now. It should be multi-column at 965px + 1 (which I'm not liking too much) and the padding should be better between the X and text.

**Pull request merged**